src/main/java/com/acme/dbo/txlog/Facade.java

```diff
@@ -1,11 +1,13 @@
 package com.acme.dbo.txlog;
 
+import static com.acme.dbo.txlog.ConsoleOutput.sysOut;
+
 public class Facade {
     public static void log(int message) {
-        System.out.println("primitive: " + message);
+        sysOut(message);
```

Comment:
ok if you want make reusable part of output. What about part of decorating (prefixing) message?

Reply:
oh, added decorator